node/src/chain_specs/manta/mainnet_genesis.rs

```diff
@@ -0,0 +1,212 @@
+        ), // CONTROLLER 1 ss58 12gWDW1z7T1AN1Vm86RcvRdSmyjtJ1Mv4LHmYvmq9LJG8czh
+        (
+            hex!("a4c2839433ae05aa48e615e5a6917cc98e4a2886272f2519347f2be7c0c6853d").into(),
+            INITIAL_CONTROLLER_BALANCE,
+        ), // CONTROLLER 2 ss58 14j2dssnCyQD2xWezTvZmbZPSKDBrwKcAcBtXrE6XgeGfzGx
+        (
+            hex!("3e31eaf98e0f6dd9b088c095757fac00dc9e3fe6af29e42caea94f7d76dd3444").into(),
+            INITIAL_CONTROLLER_BALANCE,
+        ), // CONTROLLER 3 ss58 12QYoFyrrEomSq4tMPxVxnWvPwVVKQHUpDWZuDxcg5Gv3c9E
+        (
+            hex!("0619e4018531059b677d70b476b72431d3e88fe71d2c27bba5d945975e824956").into(),
+            INITIAL_CONTROLLER_BALANCE,
+        ), // COLLATOR 1
+        (
+            hex!("0202d1760e33f60f977fcff88b611cb95e66d0b3b74c12ac6c8037b8b9764a31").into(),
+            INITIAL_COLLATOR_BALANCE,
+        ), // COLLATOR 2
+        (
+            hex!("4ece9156edd66af83b0c6afe17878844d873a08fd05bd9dbc17f9712814e9952").into(),
+            INITIAL_COLLATOR_BALANCE,
+        ), // COLLATOR 3
+        (
+            hex!("7a5cef43508640b6839c37a41272dec83dda4df28503bd87436c197d213f692c").into(),
+            INITIAL_COLLATOR_BALANCE,
+        ), // COLLATOR 4
+        (
+            hex!("4039a894994d326e32801a0fc19a267383b6231e74bc9c3fd1021d949d849406").into(),
+            INITIAL_COLLATOR_BALANCE,
+        ), // COLLATOR 5
+        (
+            root_key.clone(),
+            TOTAL_ISSUANCE - 4 * INITIAL_CONTROLLER_BALANCE - 4 * INITIAL_COLLATOR_BALANCE,
+        ), // SUDO ACCOUNT
+    ];
+    #[allow(clippy::assertions_on_constants)]
+    const _: () = assert!(
+        NORMAL_COLLATOR_MINIMUM_STAKE < INITIAL_COLLATOR_BALANCE,
+        "won't be able to register collator, balance in account set too low"
+    );
+
+    GenesisConfig {
+        system: manta_runtime::SystemConfig {
+            code: manta_runtime::WASM_BINARY
+                .expect("WASM binary was not build, please build it!")
+                .to_vec(),
+        },
+        balances: manta_runtime::BalancesConfig {
+            balances: endowments,
+        },
+        // empty aura authorities, collators registered with parachain staking instead
+        aura: Default::default(),
+        sudo: manta_runtime::SudoConfig {
+            key: Some(root_key),
+        },
+        parachain_staking: ParachainStakingConfig {
+            candidates: genesis_collators
+                .iter()
+                .map(|collator| (collator.acc.clone(), NORMAL_COLLATOR_MINIMUM_STAKE))
+                .collect(),
+            delegations: vec![],
+            // set to 0 inflation at genesis
+            inflation_config: manta_runtime::staking::inflation_config::<manta_runtime::Runtime>(),
+        },
+        parachain_info: manta_runtime::ParachainInfoConfig {
+            parachain_id: MANTA_PARACHAIN_ID.into(),
+        },
+        collator_selection: manta_runtime::CollatorSelectionConfig {
+            invulnerables: vec![],
```

Comment:
Reason to not make `genesis_collators` invulnerables

Reply:
collators are managed by `parachain_staking`

Reply:
Ah yeah, I looked at diff between calamari genesis.

Invulnerables are unneeded because we will have staking enabled from the get-go on Manta